### ISSUE TITLE
training tricks

### ISSUE THREAD

**Issue by YZY-stack** [cfzd/Ultra-Fast-Lane-Detection#172]
any training tricks?

thanks :)

**Comment by cfzd** [comment]
@YZY-stack 
I don't know the meaning of your question. Do you want to make the model better or you have problems in training?

**Comment by YZY-stack** [comment]
I want to know how to set params to achieve the best training effect like your paper because  I am training my own dataset. And I just wonder how do u adjust params or use some tricks like finetune according to your model during your training process?

**Comment by YZY-stack** [comment]
Thank you for your reply, BTW

**Comment by cfzd** [comment]
@YZY-stack 
I think you can tune the `learning rate` and `batch size` with other hyperparameters unchanged. When you get a stable result after tuning the `learning rate` and `batch size`, you can try to adjust the `griding_num`.

**Comment by YZY-stack** [comment]
@cfzd Thank u so much~
